base_import_async/models/base_import_async.py

```diff
@@ -30,7 +30,9 @@
 from openerp.models import fix_import_export_id_paths
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job, related_action
+from openerp.addons.connector.queue.job import (
```

Comment:
This will break although you don't have installed the modules. You might make by error accessible this addon in addons_path, but not having `connector` module, and this will break your Odoo installation.